modules/openy_features/openy_prgf/modules/openy_prgf_teaser/config/install/crop.type.crop_260_220.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+label: 'Crop 260×220'
+id: crop_260_220
+description: ''
```

Comment:
description for content managers needed, please

Reply:
description added